src/composables/useFlawModel.ts

```diff
@@ -44,18 +39,21 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), emit: FlawEmitt
   const osimLink = computed(() => getFlawOsimLink(flaw.value.uuid));
 
   async function createFlaw() {
-    postFlaw(flaw.value)
+    isSaving.value = true;
+    await postFlaw(flaw.value)
       .then(createSuccessHandler({ title: 'Success!', body: 'Flaw created' }))
       .then((response: any) => {
         router.push({
-          name: 'flaw-detail',
-          params: { id: response?.uuid },
+          name: 'flaw-details',
+          params: { id: response?.cve_id || response?.uuid },
         });
       })
       .catch(createCatchHandler('Error creating Flaw'));
+    isSaving.value = false;
```

Comment:
Not sure about this, but I'm thinking on the proper way to ensure `isSaving = false` is set in every scenario.
```suggestion
      .catch(createCatchHandler('Error creating Flaw'))
      .finally(() => { isSaving.value = false; });
```

Reply:
Yes, I can ditch the await above.